src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -439,16 +444,12 @@ bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
 
     // Get the expected directory that would contain hostpolicy.
     pal::string_t expected;
-    if (config.get_portable())
+    if (portable)
     {
-        if (!pal::directory_exists(fx_dir))
-        {
-            pal::string_t fx_version = specified_fx_version.empty() ? config.get_fx_version() : specified_fx_version;
-            handle_missing_framework_error(mode, config.get_fx_name(), fx_version, fx_dir, own_dir);
-            return false;
-        }
-
-        expected = fx_dir;
+        // The hostpolicy is required to be in the root framework's location
+        int root_framework = fx_definitions.size() - 1;
```

Comment:
Would it be good to define a helper method for fx_definitions.GetRootFramework() that returned `fx_definitions[fx_definitions.size() - 1]`?

Reply:
Done